controllers/categoryController.test.js

```diff
@@ -0,0 +1,124 @@
+import slugify from "slugify";
+import categoryModel from "../models/categoryModel";
+import { 
+    createCategoryController, 
+    updateCategoryController, 
+    categoryControlller, 
+    singleCategoryController, 
+    deleteCategoryCOntroller
+} from "./categoryController";
+
+jest.mock("../models/categoryModel");
+jest.mock("slugify");
+
+describe('category controller', () => {
```

Comment:
Might be better to split into different `describe` blocks. For example, one describe block for `createCategoryController` and another for `updateCategoryController` etc.